spyder/utils/stylesheet.py

```diff
@@ -396,8 +438,212 @@ def set_stylesheet(self):
             right='-3px' if WIN else '-1px'
         )
 
+        # Make scroll button icons smaller on Windows and Mac
+        if WIN or MAC:
+            css[f'QTabBar{self.OBJECT_NAME} QToolButton'].setValues(
+                padding='9px',
+            )
+
+
+class BaseDockTabBarStyleSheet(BaseTabBarStyleSheet):
+    """Base style for dockwidget tabbars."""
+
+    SCROLL_BUTTONS_BORDER_WIDTH = '2px'
+
+    def set_stylesheet(self):
+        super().set_stylesheet()
+
+        # Main constants
+        css = self.get_stylesheet()
+        self.color_tabs_separator = f'{Gray.B70}'
+
+        if is_dark_interface():
+            self.color_selected_tab = f'{QStylePalette.COLOR_ACCENT_2}'
+        else:
+            self.color_selected_tab = f'{QStylePalette.COLOR_ACCENT_5}'
+
+        # Center tabs to differentiate them from the regular ones.
+        # See spyder-ide/spyder#9763 for details.
+        css.QTabBar.setValues(
+            alignment='center'
+        )
+
+        # Style for selected tabs
+        css['QTabBar::tab:selected'].setValues(
+            color=(
+                f'{QStylePalette.COLOR_TEXT_1}' if is_dark_interface() else
+                f'{QStylePalette.COLOR_BACKGROUND_1}'
+            ),
+            backgroundColor=f'{self.color_selected_tab}',
+        )
+
+        # Make scroll button icons smaller on Windows and Mac
+        if WIN or MAC:
+            css['QTabBar QToolButton'].setValues(
+                padding='10px',
```

Comment:
This is for the new dockwidget tabs.

Reply:
Using `4px` here I got:

![image](https://github.com/spyder-ide/spyder/assets/16781833/668e3593-e45f-4b29-9bca-272c658b961b)

Reply:
To follow size with https://github.com/spyder-ide/spyder/pull/21133#discussion_r1272797950 values:

* `5px`:

![image](https://github.com/spyder-ide/spyder/assets/16781833/37e6c126-206f-4403-b887-a426dd04d471)


* `6px`:

![image](https://github.com/spyder-ide/spyder/assets/16781833/d847945c-90a0-46ea-aa5f-202d8dd37539)

Reply:
And `5px` here.